docs/overview/understanding.md

```diff
@@ -0,0 +1,328 @@
+---
+id: understanding
+title: "Understanding ZIO-Schema"
+---
+
+
+## Understanding ZIO-Schema
+
+ZIO-Schema is a part of ZIO that is used in many projects (e.g. ZIO-Flow, ZIO-SQL, etc.).
```

Comment:
I would suggest rewording this as "ZIO Schema is a library used in many ZIO projects such as ZIO Flow and ZIO SQL"